angular/projects/spark-angular/src/lib/components/inputs/sprk-radio-group/sprk-radio-group.component.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  Component,
+  AfterContentInit,
+  Input,
+  ContentChild,
+  ContentChildren,
+  QueryList,
+} from '@angular/core';
+import { uniqueId } from 'lodash';
+import { SprkRadioItemComponent } from '../sprk-radio-item/sprk-radio-item.component';
+import { SprkFieldErrorDirective } from '../../../directives/inputs/sprk-field-error/sprk-field-error.directive';
+import { SprkLabelDirective } from '../../../directives/inputs/sprk-label/sprk-label.directive';
+
+@Component({
+  selector: 'sprk-radio-group',
+  template: `
+    <div [ngClass]="getClasses()">
+      <ng-content select="[beforeSprkRadios]"></ng-content>
```

Comment:
We now allow for content to be injected before or after the radios. Before, we only allowed for after.